### DISCUSSION TITLE
Can you minimize OpenTabletDriver.Daemon.exe to the tray somehow?

### DISCUSSION THREAD

**Discussion by ghost** [OpenTabletDriver/OpenTabletDriver#1424] [Q&A] [answered]
title.
I know some apps do that with a line you can add to the shortcut (like this one https://github.com/miniant-git/REAL , you just add --tray to the shortcut) , but it doesnt work with yours

**Comment by X9VoiD** [comment, marked as answer]
Just run OTD.UX instead of running OTD.Daemon first
